modules/gathering-data-specific-features.adoc

```diff
@@ -53,7 +53,7 @@ endif::openshift-dedicated[]
 |Data collection for OpenShift Shared Resource CSI Driver.
 
 ifndef::openshift-dedicated[]
-|`registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8`
+|`registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v<installed-version-LSO>`
```

Comment:
Replaceable values should use underscores, not dashes. Can you please fix before we merge?

Reply:
Addressed